src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -162,7 +162,7 @@ public CursorsController(final CursorsService cursorsService,
     @ExceptionHandler(InvalidStreamIdException.class)
     public ResponseEntity<Problem> handleInvalidStreamId(final InvalidStreamIdException ex,
                                                          final NativeWebRequest request) {
-        LOG.warn("Stream id {} is not found", ex.getStreamId(), ex);
+        LOG.warn("Stream id {} is not found", ex.getStreamId(), ex.getMessage());
```

Comment:
@lmontrieux 
Wow... Strange. I think in this manner you need to add addional `{}` to log message.

Reply:
088ddfd